### PR TITLE
[POD-380] Continue with 2SV

### PR DESCRIPTION
**Pull request by ashwinsurendran92**
Continue with authentication by submitting the code along with the nonce.

### CODE REVIEW

**Review by prateek-citrix — APPROVED** [review]

File: lib/podio/client.rb [hunk 1]
```diff
@@ -72,6 +72,23 @@ def authenticate_with_credentials(username, password, offering_id=nil)
       @oauth_token
     end
 
+    def authenticate_with_second_factor(nonce, secondFactor, useRecoverycode)
+      if useRecoverycode == "false"
```
Comment (krishnakumarhegde) [inline lib/podio/client.rb:76]: variable `useRecoverycode` is binary or string ?

File: lib/podio/client.rb [hunk 2]
```diff
@@ -72,6 +72,23 @@ def authenticate_with_credentials(username, password, offering_id=nil)
       @oauth_token
     end
 
+    def authenticate_with_second_factor(nonce, secondFactor, useRecoverycode)
```
Comment (krishnakumarhegde) [inline lib/podio/client.rb:75]: Where are we calling this method?

File: lib/podio/client.rb (hunk 1 above)
Comment (ashwinsurendran92) [reply lib/podio/client.rb:76]: It is sent as ActionController::Parameters. Boolean values come as string.

File: lib/podio/client.rb (hunk 2 above)
Comment (ashwinsurendran92) [reply lib/podio/client.rb:75]: Will be called from fe-r.

**Review by krishnakumarhegde — APPROVED** [review]

**Pull request merged**